DashAI/back/models/scikit_learn/numerical_wrapper_for_text.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer
+
+from DashAI.back.core.schema_fields import BaseSchema, component_field, int_field
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.models.scikit_learn.sklearn_like_model import SklearnLikeModel
+from DashAI.back.models.text_classification_model import TextClassificationModel
+
+
+class NumericalWrapperForTextSchema(BaseSchema):
+    """
+    NumericalWrapperForText is a metamodel that allows text classification using
+    tabular classifiers and a tokenizer.
+    """
+
+    tabular_classifier: component_field(
+        description="Tabular model used as the underlying model"
+        "to generate the text classifier.",
+        parent="TabularClassificationModel",
+    )
+    ngram_min_n: int_field(
+        description="Minimum n_gram to use in the vectorizer.",
+        default=1,
+        ge=1,
+    )
+    ngram_max_n: int_field(
+        description="Maximum n_gram to use in the vectorizer.",
+        default=1,
+        le=1,
+    )
+
+
+class NumericalWrapperForText(TextClassificationModel, SklearnLikeModel):
+    """Text classification meta-model."""
+
+    SCHEMA = NumericalWrapperForTextSchema
+
+    def __init__(self, **kwargs) -> None:
+        self.classifier = kwargs["tabular_classifier"]
+        self.vectorizer = CountVectorizer(
```

Comment:
Maybe later we will be able to parameterize this component as well?

Reply:
Perhaps, taking into account that the model is always bag of words, we could give it a more appropriate name.

BagOfWordsTextClassificationModel or CountVectorizerTextClassificationModel?

That said, perhaps later on we could have meta-information on the models where we give them a more human name.

Reply:
I think it is better if the model has the same name on the back and front, if the name is different it might be more difficult to answer issues in the future.

Reply:
That said i think it's a great idea to give a more accurate name to the model.